boards/arm/thingy53_nrf5340/Kconfig.defconfig

```diff
@@ -82,6 +82,25 @@ config REGULATOR
 
 endif # !TRUSTED_EXECUTION_SECURE
 
+if BOARD_ENABLE_USB_CDC
+
+config USB_DEVICE_STACK
+	default y
+
+config USB_CDC_ACM
+	default y
+
+config UART_CONSOLE
+	default CONSOLE
+
+config USB_DEVICE_INITIALIZE_AT_BOOT
+	default y
+
+config SHELL_BACKEND_SERIAL_CHECK_DTR
+	default SHELL
+
```

Comment:
@nordic-krch can you investigate this issue since it will need to be resolved so this can be enabled on thingy53? https://github.com/zephyrproject-rtos/zephyr/pull/50594 can